BitwardenShared/UI/Auth/VaultUnlockSetup/VaultUnlockSetupState.swift

```diff
@@ -58,6 +62,10 @@ struct VaultUnlockSetupState: Equatable {
                     Localizations.unlockWith(Localizations.faceID)
                 case .touchID:
                     Localizations.unlockWith(Localizations.touchID)
+                case .opticID:
+                    Localizations.unlockWith(Localizations.opticID)
+                case .biometrics:
+                    Localizations.unlockWith(Localizations.biometrics)
```

Comment:
🎨 When this shows up on the Vault Unlock Setup screen, it produces the sentence "Unlock with Biometrics", and I'm not sure that's how we want to capitalize that:
<img width="350" alt="Screenshot 2024-12-31 at 11 08 05 AM" src="https://github.com/user-attachments/assets/6c5f5923-f40e-4c81-bbf6-3c27fee9699f" />